src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -136,10 +140,14 @@ export default {
 			newAcctTerms: false,
 			newsConsent: false,
 			showSsoTerms: false,
+			passwordless: false,
 		};
 	},
 	computed: {
 		registrationMessage() {
+			if (this.passwordless) {
+				return '';
+			}
```

Comment:
I don't think we can bypass any of this...it looks related to the validations we need to have in place...Also, we won't be able to control the captcha aspect here so that def needs to stay in place...

Reply:
reinstated / done